Documentation/spec/docfx_document_schema.md

```diff
@@ -0,0 +1,197 @@
+# DocFX Document Schema Design Specification
+
+## 1. Introduction
```

Comment:
>Introduction [](start = 6, length = 12)

I think we should point out that: this schema is designed for structured document, such as xml, json, yml...